notebooks/logistic_regression.ipynb

```diff
@@ -154,10 +153,10 @@
    ],
    "source": [
     "start = time.time()\n",
-    "hmc_states = fori_collect(tqdm.trange(num_samples), sample_kernel, hmc_state,\n",
+    "hmc_states = fori_collect(num_samples, sample_kernel, hmc_state,\n",
     "                          transform=lambda state: {\"coefs\": state.z[\"coefs\"],\n",
     "                                                   \"num_steps\": state.num_steps},\n",
-    "                          use_prims=False)\n",
+    "                          progbar=True)\n",
```

Comment:
this won't take effect unless you also make a similar change to `fori_collect` (removing `use_prims` arg)

Reply:
Done.